components/sidebar/unread_channel_indicator/index.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
What purpose does this file serve?

Reply:
We had a discussion last week about moving components into their own folders when they started including multiple files like snapshots and tests and such, so I was doing that here. This was my first instinct to keep the import path simple for a non-connected component, but I'm 0/5 about whether the component lives in index in that case. This is kind of nice thoughsince it keeps the file named after the component instead of having to look for `unread_channel_indicator/index.tsx` in your editor at least.